tests/test_wsireader.py

```diff
@@ -1471,13 +1721,15 @@ def test_command_line_jp2_slide_thumbnail_file_not_supported(_sample_jp2, tmp_pa
     assert isinstance(slide_thumb_result.exception, FileNotSupported)
 
 
+# @pytest.mark.skip(reason="working, skip to run other test")
 def test_openslide_read_rect_edge_reflect_padding(_sample_svs):
     """Test openslide edge reflect padding for read_rect."""
     wsi = wsireader.OpenSlideWSIReader(_sample_svs)
     region = wsi.read_rect((-64, -64), (128, 128), pad_mode="reflect")
     assert 0 not in region.min(axis=-1)
 
 
+# @pytest.mark.skip(reason="working, skip to run other test")
```

Comment:
isn't it better to remove all the commented `# @pytest.mark.skip(reason="working, skip to run other test")` lines before each testing function now that all tests are being performed? I guess we can avoid using this technique in the future as we can use the unit testing feature of our IDEs (like what @vqdang found on [VSCode](https://code.visualstudio.com/docs/python/testing)).

Reply:
being uncomment means it will run normally, you can check the test report to see nothing is skip.

Reply:
Anyways, I removed them in https://github.com/TIA-Lab/tiatoolbox/pull/123/commits/46566bb05194e308b729a0c67e21a4d8dd993ea2